extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -77,7 +62,7 @@ class WikiaSearchController extends WikiaSpecialPageController {
 	 * Handles dependency-building and special page routing before calling controller actions
 	 */
 	public function __construct() {
-        // note: this is required since we haven't constructed $this->wg yet
+		// note: this is required since we haven't constructed $this->wg yet
 		global $wgWikiaSearchIsDefault;
 		$specialPageName = $wgWikiaSearchIsDefault ? 'Search' : 'WikiaSearch';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Search" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)